src/components/expanded-state/UniqueTokenExpandedState.tsx

```diff
@@ -305,6 +324,22 @@ const UniqueTokenExpandedState = ({
     usePersistentDominantColorFromImage(asset.lowResUrl).result ||
     colors.paleBlue;
 
+  const setAccentColor = useSetRecoilState(accentColorAtom);
```

Comment:
why are we setting all this logic in this component? should we not do it more declaratively? just worried if we were to use the edit profile feature in other places, then we will have to remember to copy/paste or compose this code as well.

Reply:
fair enough, passing that data through route params now